src/main/java/it/pagopa/pn/client/b2b/pa/utils/TimingForTimeline.java

```diff
@@ -11,16 +12,37 @@
 
 @Component
 public class TimingForTimeline {
-
     private final PnB2bClientTimingConfigs timingConfigs;
-    public record TimingResult(int numCheck,int waiting) { }
+    public record TimingResult(int numCheck, int waiting) { }
+
+    @Value("${pn.config.timing.timeline.tuning-value}")
+    private Integer tuningValue;
 
     @Autowired
     public TimingForTimeline(PnB2bClientTimingConfigs timingConfigs){
         this.timingConfigs = timingConfigs;
     }
 
 
+    public TimingResult getTimingForElement(String element, boolean isSlow){
```

Comment:
aggiungere overload del metodo per evitare che altri chiamanti vadano in errore

Reply:
L'overload del metodo è stato aggiunto come da richiesta